src/HubSpot.Client/Authentication/OAuthHubSpotAuthenticator.cs

```diff
@@ -110,7 +108,7 @@ public AuthToken(string token, DateTimeOffset expiresOn)
         }
     }
 
-    public class OAuthOptions
+    public class OAuthSettings
     {
         public string ClientId { get; set; }
 
```

Comment:
I think we can move one of these Settings/Options into the Hubspot RA, since it was mainly in use for that app